config/rbac/kustomization.yaml

```diff
@@ -11,6 +11,9 @@ resources:
 - builder_role_binding.yaml
 - leader_election_role.yaml
 - leader_election_role_binding.yaml
+# Uncomment the following 2 lines if you want to deploy on OpenShift
```

Comment:
As I understand, this is not needed anymore.

Reply:
Done ;-)